skins/cat17/faq-page/src/Faq.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+ <div id="faq" class="container">
+ 	<h2>Häufige Fragen</h2>
```

Comment:
Remember to make this translatable. If this makes the 5-point task too big or you don't consider this to be part of the ticket, please create a new ticket so we don't forget.

Reply:
I thought this will be part of the task to load the actual content of the page :thinking: